lighthouse-core/test/audits/dobetterweb/uses-http2-test.js

```diff
@@ -61,4 +61,22 @@ describe('Resources are fetched over http/2', () => {
       assert.ok(auditResult.displayValue === '');
     });
   });
+
+  it('results are correct when some requests are handled by service worker', () => {
+    const clonedNetworkRecords = JSON.parse(JSON.stringify(networkRecords));
+    clonedNetworkRecords.forEach(record => {
+      // convert http 1.x to service worker requests
+      if (record.protocol === 'http/1.1') {
+        record.fromServiceWorker = true;
+      }
+    });
+
+    return UsesHTTP2Audit.audit(
+      getArtifacts(clonedNetworkRecords, URL), {computedCache: new Map()}
+    ).then(auditResult => {
+      assert.equal(auditResult.rawValue, false);
+      assert.ok(auditResult.displayValue.match('1 request not'));
+      expect(auditResult.details).toMatchSnapshot();
```

Comment:
I do ❤️ me some `toMatchSnapshot` :D 

Since this test doesn't really have any impact on anything but the items though, I'd want to limit it to that.

Aside, up to this point, all uses of snapshot testing expect for 1 are uses where there's arguably not a permanent correct answer and/or we update the values frequently. When it's essentially used in place of `.toEqual`, I think it should be kept to `InlineSnapshot`. I believe @brendankenny feels differently about these sorts of uses though 😉

Reply:
I remove the snapshot as it wasn't a big deal here :)